### ISSUE TITLE
False positives with rule 932200 when sending json payloads

### ISSUE THREAD

**Issue by mic345** [coreruleset/coreruleset#2607]
### Description

Unfortunately we can not share the log for privacy reasons. Here is some information though:

1. A link to a [101regex demonstrating the false positive](https://regex101.com/r/m7yfiC/1).
2. The data is sent via a `POST` request through the request body.
3. The `Content-Type` is `application/json`

### Audit Logs / Triggered Rule Numbers

932200

### Your Environment

* CRS version: 3.3.2
* ModSecurity version: 2.9.5
* Web Server: Apache 2.4.53
* OS: Linux 5.10.96-90.460.amzn2.x86_64

### Confirmation

[ ✔️] I have removed any personal data (email addresses, IP addresses,
    passwords, domain names) from any logs posted.

Regular expression from rule:
```
([*?`\\'][^\/\n]+\/|\$[({\[#a-zA-Z0-9]|\/[^\/]+?[*?`\\'])
```

Matched string:
```
{"content":"{\"value\":\"https://abcabcabc.cloudfront.net/cdn-images.mysite.com/abcabcabc
\",\"lastModified\":1653822419828}"}
```

The current version of the regular expression (v4.0/dev) also matches the string.

**Comment by theseion** [comment]
There's not much we can do here. Transporting JSON inside JSON is an application specific case. I suggest that you disable the rule dynamically for JSON requests. If there's a way to identify the requests that cause this, e.g., by header, or a special value in the payload, I would restrict deactivation of the rule to those requests.

To disable a rule dynamically, you would use `ruleRemoveTargetById` (see https://github.com/SpiderLabs/ModSecurity/wiki/Reference-Manual-(v2.x)#ctl).

**Comment by mic345** [comment]
👍

This is what we did, thanks for the feedback!

**Comment by theseion** [comment]
Great. I'll close this issue then.
